package.json

```diff
@@ -161,70 +161,83 @@
       "default": "pdf",
       "order": 10
     },
+    "producer": {
+      "description": "Program to use when post-process DVI output in order to produce PDF. This is not used for pdflatex based builders. Currently only supported for latexmk.",
+      "type": "string",
+      "enum": [
+        "default",
```

Comment:
What is default? Can we possibly be more opinionated and explicitly pick an explicit default producer instead?